hello-feature-module/build.gradle

```diff
@@ -24,7 +24,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-alpha1'
+        classpath 'com.android.tools.build:gradle:3.0.0-alpha9'
```

Comment:
Next time, please keep your Merge Request as simple as possible. Nowadays, it doing many things like: fix exception, updating classpath and compile Sdk  instead of one declared in MR title. That's a bit weird and misleading - think of creating a few smaller separate MRs.